docs/general-terminology/open-source.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: Open Source
+edit: https://github.com/pbclife/gitopener.vercel.app/edit/main/docs/open-source.mdx
```

Comment:
Remove this edit key and link the link will be auto generated, in the beginning I was doing it explicitly but in new updates I made it automatic